GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionGroupRestService.java

```diff
@@ -141,6 +141,17 @@
                 }
             }
         }
+
+        // in this case, we are just trying to get a single question group
```

Comment:
I don't understand the purpose for this. Why not use the [question_groups/<id>](https://github.com/akvo/akvo-flow/blob/feature/857-questiongroup-copy/GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionGroupRestService.java#L161) endpoint?